apps/glusterfs/allocator_simple.go

```diff
@@ -10,8 +10,9 @@
 package glusterfs
 
 import (
-	"github.com/boltdb/bolt"
 	"sync"
+
+	"github.com/boltdb/bolt"
```

Comment:
Editor again, standard packages and github packages are separated by a space and standard packages should come first. Trying to revert this change.